src/types/index.ts

```
@@ -15,6 +15,7 @@ export interface PayloadData {
 export interface AuthenticatedRequest<Params = Record<string, never>, Query = ParsedQs, Body = Record<string, never>>
   extends Request<Params, any, Body, Query> {
   payload?: PayloadData;
+  userId?: Number;
```

Biome (1.9.4)

[error] Don't use 'Number' as a type.

Use lowercase primitives for consistency.
Safe fix: Use 'number' instead


(lint/complexity/noBannedTypes)